src/main/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceImpl.java

```diff
@@ -208,4 +201,11 @@ public OffenderSentenceDetail getOffenderSentenceDetail(final String offenderNo)
                 .headers(offenderNumbers.getPaginationHeaders())
                 .body(offenderNumbers.getItems());
     }
+
+    @Override
+    public PrivilegeSummary getLatestBookingIEPSummaryForOffender(final String offenderNo, final boolean withDetails) {
```

Comment:
the /offenders endpoint has that information in it already BTW https://github.com/ministryofjustice/prison-api/blob/main/src/main/java/uk/gov/justice/hmpps/prison/api/model/InmateDetail.java#L175